### PR TITLE
Updated README.md for instructions to run locally

### PR DESCRIPTION
**Pull request by ElementUser**
Updated the "Running locally" section of README.md with the correct bin usage (bun)
- Added installation step for the tsx bin & updated the installation doc URL (previous URL does not work anymore)

### CODE REVIEW

**Comment by Sendouc** [comment]
Thanks, the culprit was my fast copy paste.

```
Prerequisites:
- Install latest version of [tsx](https://tsx.is/getting-started)
```

We can drop this as `npx` will install it automatically if you don't have it.

**Comment by ElementUser** [comment]
> Thanks, the culprit was my fast copy paste.
> 
> ```
> Prerequisites:
> - Install latest version of [tsx](https://tsx.is/getting-started)
> ```
> 
> We can drop this as `npx` will install it automatically if you don't have it.

Done =)

File: package.json [hunk 1]
```diff
@@ -105,8 +105,11 @@
 	},
 	"trustedDependencies": [
 		"@biomejs/biome",
+		"@swc/core",
```
Comment (ElementUser) [inline package.json:108]: I'm not sure where these come from, but they aren't in the main branch - I'll work on removing them

Comment (ElementUser) [reply package.json]: Looks proper now =)

**Pull request merged**